.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+# iDE
```

Comment:
@akram256 , Could you please rename this to `IDE`. Otherwise nice work

Reply:
Hey @Stanley-Okwii , i have worked on it. Thank you so much